packages/jaeger-ui/src/components/SearchTracePage/SearchResults/index.tsx

```diff
@@ -16,15 +16,14 @@
 
```

Comment:
This file was apparently using Flow types, so I ended up converting it to TS instead of just renaming to `.jsx`.